CONTRIBUTING.md

```diff
@@ -22,6 +22,11 @@ Every code contribution to this project is highly valued and appreciated. I enco
    - PcapPlusPlus compiles successfully on your machine, including all unit-tests and examples (just run `make` from PcapPlusPlus main directory)
    - Unit-tests pass succssfully on your machine (both `Tests/Packet++Test` and `Tests/Pcap++Test`)
    - All new APIs are well documented using Doxygen (please use @ for keywords)
+   - Make sure that pre-commit hooks are passing by using `pre-commit run --all-files`. For pre-commit hooks you need to install `cppcheck` on your system. Recommended and tested version is 2.7. You can install by using
+     - For Linux (apt) `sudo apt install cppcheck`
+     - For MacOS/Linux (brew) `brew install cppcheck@2.7`
```

Comment:
@seladb I fixed at 2.7 for now to keep simple github workflow yml. Can you check the brew command on a MacOS?

Reply:
With `homebrew` it's not easy to install an old version of a package, `brew install cppcheck@2.7` doesn't work 😕  

Maybe instead of `brew` we can just add a link to the release page: https://github.com/danmar/cppcheck/releases/tag/2.7 ?

We can do something like this:
```markdown
For pre-commit hooks you need to install `cppcheck` on your system. Recommended and tested version is 2.7. You can install by using:
- For Linux (apt) `sudo apt install cppcheck=2.7-1`
- For Windows (using `choco`): `choco install cppcheck --version=2.7`
- For Windows (MSI install): https://github.com/danmar/cppcheck/releases/download/2.7/cppcheck-2.7-x64-Setup.msi
- Build from source: https://github.com/danmar/cppcheck/releases/tag/2.7
```